REPL.cpp

```diff
@@ -79,7 +79,7 @@ llvm::Expected<std::unique_ptr<REPL>> REPL::Create(
 REPL::REPL(bool is_playground, std::string default_module_cache_path)
     : m_is_playground(is_playground),
       m_default_module_cache_path(default_module_cache_path),
-      m_curr_input_number(0),
+      m_curr_input_number(1),
```

Comment:
Why does it start at 1 now?

Reply:
Before we incremented at the start of every call to GetLine(). This is no longer the case, so GetLine() just prints the current input number as the prompt. The input number is the incremented in the ExecuteSwift function (we need to do this because it creates a `__repl<input number>_res` variable. Since playground doesn't call GetLine, this would never get incremented, and we'd end up with duplicate definitions of `__repl0_res`.